volatility3/framework/interfaces/objects.py

```diff
@@ -142,7 +151,7 @@ def get_symbol_table_name(self) -> str:
         """
         if constants.BANG not in self.vol.type_name:
             raise ValueError(f"Unable to determine table for symbol: {self.vol.type_name}")
-        table_name = self.vol.type_name[:self.vol.type_name.index(constants.BANG)]
```

Comment:
This is in a helper function that you never have to see and there's no good reason to change it.

Reply:
restored it